src/analysis/convex/basic.lean

```diff
@@ -4,94 +4,156 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Alexander Bentkamp, Yury Kudriashov
 -/
 
-import analysis.normed_space.basic
 import data.complex.basic
 import data.set.intervals
 import tactic.interactive
 import tactic.linarith
 import linear_algebra.basic
 import ring_theory.algebra
+import algebra.pointwise
 
 /-!
 # Convex sets and functions on real vector spaces
 
-In a real normed space, we define the following objects and properties.
+In a real vector space, we define the following objects and properties.
 
 * `segment x y` is the closed segment joining `x` and `y`.
-* A set `A` is `convex` if for any two points `x y ∈ A` it includes `segment x y`;
-* A function `f` is `convex_on` a set `D` if `D` is itself a convex set, and for any two points
-  `x y ∈ D` the segment joining `(x, f x)` to `(y, f y)` is (non-strictly) above the graph of `f`;
-  equivalently, `convex_on f D` means that the epigraph `{p : α × ℝ | p.1 ∈ D ∧ f p.1 ≤ p.2}`
+* A set `s` is `convex` if for any two points `x y ∈ s` it includes `segment x y`;
+* A function `f` is `convex_on` a set `s` if `s` is itself a convex set, and for any two points
+  `x y ∈ s` the segment joining `(x, f x)` to `(y, f y)` is (non-strictly) above the graph of `f`;
+  equivalently, `convex_on f s` means that the epigraph `{p : E × ℝ | p.1 ∈ s ∧ f p.1 ≤ p.2}`
   is a convex set;
 * Center mass of a finite set of points with prescribed weights.
+* Convex hull of a set `s` is the minimal convex set that includes `s`.
 
```

Comment:
You should mention the standard simplex somewhere in the file-level docstring.

Reply:
Done.